R/allocate_reference_times.R

```diff
@@ -0,0 +1,337 @@
+#' Allocate training volume based on combination of defaults and user-specified
+#'   values for training volume for delay and uncertainty estimation.
+#' @description Given the reporting triangle, the maximum delay, and
+#'    optionally the user-specified scale factor on the max delay to be used as
+#'    total reference times and the proportion of those reference times to
+#'    be used for delay estimation, allocate reference times accordingly
+#'
+#' @inheritParams estimate_delay
+#' @param scale_factor Numeric value indicating the multiplicative factor on
+#'     the maximum delay to be used for estimation of delay and uncertainty.
+#'      Default is `3`.
+#' @param prop_delay Numeric value <1 indicating what proportion of all
+#'     reference times in the reporting triangle to be used for delay
+#'     estimation. Default is `0.5`.
+#' @param size_min_retro_nowcasts Integer indicating the minimum number of
```

Comment:
Why is this called size? What does it add as a prefix?

Reply:
Can remove the reference to size and just use `n`?

Reply:
The slight distinction in my head was `n` is user-specified, `size` is internally determined